rotkehlchen/assets/unknown_asset.py

```diff
@@ -15,6 +16,22 @@
 SerializeAsDictKeys = Union[List[str], Tuple[str, ...], Set[str]]
 
 
+class FakeAsset(NamedTuple):
+    """A class that fakes Asset properties when the class can't be
+    instantiated (i.e. the identifier belongs to an unknown ethereum token).
+
+    For instance this happens with DB trades entries whose `fee_currency` is an
+    unknown ethereum token and Trade can't be instantiated.
+    """
+    identifier: str
+
+    def __str__(self) -> str:
+        return self.identifier
+
+    def to_binance(self) -> str:
```

Comment:
@LefterisJP I see this strategy as a short term solution for passing `make lint`. I think we should think about the opposite of `Asset` for "unknown ethereum tokens" if they keep clashing around.